src/IdentityServer4/test/IdentityServer.UnitTests/Validation/Secrets/SecretValidation.cs

```diff
@@ -45,7 +45,7 @@ public async Task Valid_Single_Secret()
                 Type = IdentityServerConstants.ParsedSecretTypes.SharedSecret
             };
 
-            var result = await _validator.ValidateAsync(secret, client.ClientSecrets);
```

Comment:
Why did you change the order?

Reply:
Consistency. The ISecretValidator interface uses that order.

Reply:
If you introduce a breaking change we cannot merge this for 3.1..

Reply:
In that case I can put up a new PR with the order of the arguments swapped around.